src/diff.c

```diff
@@ -1863,6 +1872,16 @@ int git_diff_commit_as_email(
 	opts.summary = git_commit_summary(commit);
 	opts.author = git_commit_author(commit);
 
+	/* The first line of the commit message is identical to
+	 * the summary and thus needs to be skipped */
+	if ((ptr = git_commit_message(commit)) != NULL) {
+		/* Find end of first line */
+		ptr = strpbrk(ptr, "\r\n");
```

Comment:
Fun fact: the second argument to `strpbrk()` is a list of chars to find, in string form (because how else would you represent a list of characters ;)